diff --git a/sys-include/math.h b/sys-include/math.h
index bc7712a..a6bdf06 100644
--- a/sys-include/math.h
+++ b/sys-include/math.h
@@ -416,7 +416,7 @@ extern __stdargs float hypotf (float, float);
    simply call the double functions.  On Cygwin the long double functions
    are implemented independently from newlib to be able to use optimized
    assembler functions despite using the Microsoft x86_64 ABI. */
-#if defined (_LDBL_EQ_DBL) || defined (__CYGWIN__)
+#if defined (_LDBL_EQ_DBL) || defined (__CYGWIN__) || defined (__amigaos__) /* AmigaChrome: long double math via double, in libos32compat */
 /* Reentrant ANSI C functions.  */
 #ifndef __math_68881
 extern __stdargs long double atanl (long double);
